main/src/ca/uwaterloo/flix/api/lsp/Visitor.scala

```diff
@@ -755,6 +752,18 @@ object Visitor {
     visitExpr(exp)
   }
 
+  private def visitExtMatchRule(rule: ExtMatchRule)(implicit a: Acceptor, c: Consumer): Unit = rule match {
+    case ExtMatchRule(_, pats, exp, loc) =>
+      if (!a.accept(loc)) {
+        return
+      }
+
+      c.consumeExtMatchRule(rule)
+
+      pats.foreach(visitExtPattern)
+      visitExpr(exp)
```

Comment:
I'd expect these to be swapped? exp first rules after

Reply:
Same as the other comment, right? Order of fields.